.github/workflows/check_smoke_tests.yml

```diff
@@ -57,7 +57,8 @@ jobs:
           elif [[ "${{ matrix.database }}" == "mysql" ]]; then
             export DB_URI="mysql+pymysql://root:pw@localhost:3306/mysql"
           elif [[ "${{ matrix.database }}" == "sqlserver" ]]; then
-            export DB_URI="mssql+pyodbc://sa:YourStrong!Passw0rd@localhost:1433/master?driver=ODBC+Driver+18+for+SQL+Server&TrustServerCertificate=yes"
+            export DB_URI="mssql+pyodbc://sa:YourStrong!Passw0rd@localhost:1433/master?driver=FreeTDS&TrustServerCertificate=yes"
```

Comment:
I was thinking the driver should continue to match what we have documented in docs/site/app-configuration.md.  That way we know what we are telling people to use still works.  Thoughts?

Reply:
You're totally right